scripts/generate-schematic-imports-map.ts

```diff
@@ -0,0 +1,53 @@
+import {sync as glob} from 'glob';
+import {readFileSync, writeFileSync} from 'fs';
+import {join, basename} from 'path';
+import * as ts from 'typescript';
+
+// Script that generates mappings from our publicly-exported symbols to their entry points. The
+// mappings are intended to be used by the secondary entry points schematic and should be committed
+// next to the relevant schematic file.
+// Can be run using `ts-node --project scripts scripts/generate-schematic-imports-map.ts`.
+const mappings: {[symbolName: string]: string} = {};
+const outputPath = join(__dirname, '../temp-entry-points-mapping.json');
+
+glob('**/*.d.ts', {
+  absolute: true,
+  cwd: join(__dirname, '../tools/public_api_guard/material')
```

Comment:
I don't think we have any validation that ensures we have goldens for every package (yet)? Can we be sure that we did not miss any package entry-point?

Reply:
I don't think we can be absolutely sure that we haven't missed anything, but I did a manual check a while ago and the only package I found that was missing was `cdk/portal` that I did a PR for.